modules/apps/portal-workflow/portal-workflow-kaleo-service/service.xml

```diff
@@ -895,8 +895,8 @@
 		<!-- Audit fields -->
 
 		<column name="companyId" type="long" />
-		<column name="userId" type="long" />
-		<column name="userName" type="String" />
+		<column name="userId" type="long" uad-anonymize-field-name="userId" />
+		<column name="userName" type="String" uad-anonymize-field-name="fullName" />
```

Comment:
@IstvanD would be needed to add the `uad-anonymize-field-name` property for completionUserId column?

Reply:
hi @MarinhoFeliphe because the `completionUserId` column is part of KaleoUADConstants.USER_ID_FIELD_NAMES_KALEO_TASK_INSTANCE_TOKEN, `completionUserId` will be accounted for processing in the `autoAnonymize` method.